backintime-compiler/src/main/kotlin/com/github/kitakkun/backintime/compiler/fir/checkers/KtDefaultErrorMessagesBackInTime.kt

```diff
@@ -4,6 +4,7 @@ import org.jetbrains.kotlin.diagnostics.KtDiagnosticFactoryToRendererMap
 import org.jetbrains.kotlin.diagnostics.rendering.BaseDiagnosticRendererFactory
 
 object KtDefaultErrorMessagesBackInTime : BaseDiagnosticRendererFactory() {
+    @Suppress("ktlint:standard:property-naming")
     override val MAP = KtDiagnosticFactoryToRendererMap("BackInTime").apply {
```

Comment:
can't be fixed because `MAP` is a property from a class inside kotlin official library.